help/configure-default-new-user-settings.md

```diff
@@ -39,6 +42,22 @@ preference settings, including the following:
 
 {end_tabs}
 
+## Configure default language for new users
+
+Your organization's [language](/help/configure-organization-language) will be
+the default language for new users.
```

Comment:
I think it's only the default language for new users who are created through an automatic process -- I'm pretty sure we get a language from the user's browser for other users?

Reply:
Good catch! Fixed and updated the screenshot above.